### PR TITLE
chore: add no unused imports biome rule

### PR DESCRIPTION
**Pull request by nunogois**
Adds a Biome rule for "no unused imports", which is something we sometimes have trouble catching.

We're adding this as a warning for now. It is safely and easily fixable with `yarn lint:fix`.

image
image

### CODE REVIEW

File: biome.json [hunk 1]
```diff
@@ -4,9 +4,6 @@
     "enabled": true,
     "rules": {
       "recommended": true,
-      "performance": {
-        "noDelete": "off"
-      },
```
Comment (nunogois) [inline biome.json:9]: This was a duplicate property. See `performance` below. I kept the one that has the most options set.

**Review by sighphyre — APPROVED** [review]
Phew that's a lot of used imports! Nice one!

**Review by chriswk — APPROVED** [review]
Nice, is unusedImport now a safe fix? It didn't use to be.

**Comment by nunogois** [comment]
> Nice, is unusedImport now a safe fix? It didn't use to be.

The fix itself is safe, but it's still part of the nursery set of rules: https://biomejs.dev/linter/rules/no-unused-imports/

**Pull request merged**